src/styles/catalog.scss

```diff
@@ -0,0 +1,21 @@
+.catalog {
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  justify-items: center;
+  justify-content: center;
```

Comment:
Using both 'justify-items' and 'justify-content' on the same grid container might be redundant. 'justify-items' aligns the items within their grid areas, while 'justify-content' aligns the grid within its container. Consider if both are necessary for your layout.